clang/include/clang/Driver/Options.td

```diff
@@ -6345,6 +6345,10 @@ def J : JoinedOrSeparate<["-"], "J">,
   Group<gfortran_Group>,
   Alias<module_dir>;
 
+def no_fortran_main : Flag<["-"], "fno-fortran-main">,
+  Visibility<[FlangOption]>, Group<f_Group>,
+  HelpText<"Don't link in Fortran main">;
```

Comment:
[nit] For folks less familiar with the compiler internals, it would be a good to be a bit more specific, e.g:

```bash
Skip Fortran_main.a (provided by Flang) when linking
```

WDYT? This is also something worth documenting ;-)

Reply:
Done, with a slightly different wording